app/models/deed.rb

```diff
@@ -19,4 +19,7 @@ def parent_product
     end
   end
 
+  def self.deeds_of_user(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `deeds_of_user(*)` if you want the method to accept any arguments but don't care about them.